packages/prisma-generator/tests/type-compatibility/expected/unsanitized.ts

```diff
@@ -0,0 +1,592 @@
+import { type DataType, type SchemaType } from '@genseki/prisma-generator'
+
+export interface DataModelConfig {
+  name: 'DataModel'
+  dbModelName: 'Data'
+  prismaModelName: 'Data'
+  primaryFields: ['id']
+  uniqueFields: [['id']]
+}
+
+export interface DataModelShape {
+  id: {
+    schema: typeof SchemaType.COLUMN
```

Comment:
Should this key name be `fieldType` to represent `SchemaType` ?

Reply:
There's so many type starting with `Field`. I would like to make this different so I decided to use `Schema`